basics/data-import/pinot-file-system/import-from-hdfs.md

```diff
@@ -14,18 +14,18 @@ You can enable the [Hadoop DFS](https://hadoop.apache.org/) using the plugin `pi
 By default Pinot loads all the plugins, so you can just drop this plugin there. Also, if you specify `-Dplugins.include`, you need to put all the plugins you want to use, e.g. `pinot-json`, `pinot-avro` , `pinot-kafka-2.0...`
 {% endhint %}
 
-HDFS implementation provides the following options -
+HDFS implementation provides the following options:
 
-* `hadoop.conf.path` : Absolute path of the directory containing hadoop XML configuration files such as **hdfs-site.xml, core-site.xml** .
+* `hadoop.conf.path` : Absolute path of the directory containing hadoop XML configuration files, such as **hdfs-site.xml, core-site.xml** .
 * `hadoop.write.checksum` : create checksum while pushing an object. Default is `false`
 * `hadoop.kerberos.principle`
 * `hadoop.kerberos.keytab`
 
 Each of these properties should be prefixed by `pinot.[node].storage.factory.class.hdfs.` where `node` is either `controller` or `server` depending on the config
 
-The `kerberos` configs should be used only if your Hadoop installation is secured with Kerberos. Please check [Hadoop Kerberos guide ](https://docs.cloudera.com/documentation/enterprise/5-8-x/topics/cdh\_sg\_kerberos\_prin\_keytab\_deploy.html#topic\_3\_4)on how to generate Kerberos security identification.
+The `kerberos` configs should be used only if your Hadoop installation is secured with Kerberos. Refer to the [Hadoop Kerberos guide ](https://docs.cloudera.com/documentation/enterprise/5-8-x/topics/cdh\_sg\_kerberos\_prin\_keytab\_deploy.html#topic\_3\_4) for information on how to generate Kerberos security identification.
```

Comment:
```suggestion
The `kerberos` configs should be used only if your Hadoop installation is secured with Kerberos. For information on how to generate Kerberos security identification, see [Hadoop Kerberos guide](https://docs.cloudera.com/documentation/enterprise/5-8-x/topics/cdh\_sg\_kerberos\_prin\_keytab\_deploy.html#topic\_3\_4).
```

Reply:
this link returns a 404, maybe a more generic, see Hadoop Kerberos documentation?